feat(synthetics): Removed old IP and inserted new IP ranges, new call out, new metadescription (#6302)
newrelic/docs-website: src/content/docs/synthetics/synthetic-monitoring/administration/synthetic-public-minion-ips.mdx

  - /docs/synthetics/synthetic-public-minion-ips
---

New Relic uses public minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe and are in charge of running your monitors.

> Because of this, ensure your firewall allows synthetics network requests. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.

Reviewer: Should this be `synthetic network request`?

Reviewer: Genuinely not sure what the terminology we use here is.

> Keep in mind that:

Reviewer: Try to be more prescriptive here. A good rule of thumb is to avoid "You can" or "You should" when recommending options. Maybe rephrase this optional steps or something similar.

Author: Went with optional steps. When I redo this doc, I'm going to revisit this tbh.

  - /docs/synthetics/synthetic-public-minion-ips
---

> You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed from different data centers around the globe and are in charge of running your monitors.

Reviewer: ```suggestion
You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed from different data centers around the globe and run your monitors.
```

Author: How do we feel about: 

> You can set up minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions run your monitors and are deployed from different data centers around the globe.

Reviewer 2: The original wording is less confusing to me. When we mention that customers can set up minions it will cause confusion between public and private minions. We create and maintain minions for our public locations where customers are responsible for deploying minions only for private locations behind the firewall.

```suggestion
New Relic maintains public minions to execute your [synthetic monitors](/docs/synthetics/new-relic-synthetics/using-monitors/add-edit-monitors). These minions are deployed in different data centers around the globe and are in charge of running your monitors.
```

> Because of this, ensure your firewall allows your minions to make network requests through it. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.

Reviewer: Where can users find these publicly available addresses? If it's in this doc, link over to it.

Reviewer 2: We will cause confusion between public and private minions if we refer to these as your minions. 
```suggestion
Because of this, ensure your firewall allows synthetics network requests. Minion IP addresses are publicly available in JSON format, so you can easily read and parse them.
```

## IP addresses are not personal data [#not-personal-data]

> Minions are deployed on servers, and the agents are activated using non-personal credentials. IP addresses associated with minions running on servers are not personal data under data protection and privacy laws. For more information, see the [Synthetic's security documentation](/docs/synthetics/new-relic-synthetics/getting-started/security-new-relic-synthetics).

Reviewer: This wording can change the meaning pretty significantly. Do the users provide the credentials? If so, I would probably keep "expected to be" as it implies the best practice is to use non-personal credentials. If we control the credentials, this edit is fine.

Author: !! Good catch! I'll change it back to 'expected to be.' I removed it to streamline the sentence and didn't realize meaning would be affected ;)

## Daily JSON listings for IP addresses [#dynamic-ip-addresses]

IP addresses for released locations are subject to change. If a change is needed, we'll attempt to proactively notify customers prior to any changes via e-mail. You can also check the [Explorers Hub](https://discuss.newrelic.com/) for updates.

<Callout variant="important">

>     By the end of March 2022, [new IP address ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips) may require that you reconfigure your firewall settings.

Reviewer: I'd get as granular as you can here. This is fine for now, but I'd update it to the exact date as soon as it's known.

Reviewer 2: We don't want to remove this callout about production vs eu in the JSON listings. While the new IP address ranges will include both our US and EU locations, the existing IPs will continue to be used for months too and this production / eu breakdown is important for those IPs.

>     By the end of March 2022, [new IP address ranges](/docs/synthetics/synthetic-monitoring/administration/new-synthetic-public-minion-ips) may require that you reconfigure your firewall settings.

Reviewer: I'd also be a bit more specific about what would entail "may" becoming "will". I'd specify in which instances someone would have to reconfigure.

Author: "May" was to account for the short period that both runtimes would be supported for already-existing monitors. Users won't have to update their allow lists until:

- They create a new monitor after the new runtime launch
- The old runtime (and their associated / old IPs) is no longer supported. (No date for this yet) 

At that point, they'll have to update their allow lists. The call out is mostly just to notify users that this is in the works to keep an eye out for.
